lib/win32/taskscheduler.rb

```diff
@@ -39,10 +39,10 @@ class Error < StandardError; end
     TASK_EVENT_TRIGGER_ON_IDLE = 5
 
     # Trigger is set to run the task at system startup
-    TASK_EVENT_TRIGGER_AT_SYSTEMSTART = 6
+    TASK_EVENT_TRIGGER_AT_SYSTEMSTART = 7
 
     # Trigger is set to run the task when a user logs on
-    TASK_EVENT_TRIGGER_AT_LOGON = 7
+    TASK_EVENT_TRIGGER_AT_LOGON = 8
```

Comment:
Why are these changing? That doesn't match the documented values from Microsoft.

https://msdn.microsoft.com/en-us/library/windows/desktop/aa383620(v=vs.85).aspx

Reply:
Hi @btm ,
While testing previous trigger value, `with user_id`, task for `log on for 'any user'` was created.
On further analysis, we found that its actual value should be 9  ( Ref: `Task Scheduler 2.0 Enumerated Types` : https://msdn.microsoft.com/en-us/library/windows/desktop/aa383915(v=vs.85).aspx )

As per the current library code, `it increments the constant by 1` and then creates required task !
Also, while displaying, first `index value is incremented by 1`, and then trigger's value is returned.
( Ref while creation: https://github.com/chef/win32-taskscheduler/blob/ole/lib/win32/taskscheduler.rb#L644:L663
 Ref while display:  https://github.com/chef/win32-taskscheduler/blob/ole/lib/win32/taskscheduler.rb#L800 )
Hence, we assigned `TASK_EVENT_TRIGGER_AT_LOGON = 8`

But I think we may use actual constants value without any incrementation in any of the methods. Could you please let us know your views about this